src/delimiter.js

```diff
@@ -375,9 +375,19 @@ const makeStackedDelim = function(
 
     // To cover the gap create by the overlaps, insert one more repeat element,
     // at a position that juts 0.005 above the bottom of the top element.
-    inners.push({type: "kern", size: shiftOfExtraElement});
-    inners.push(makeInner(repeat, font, mode));
-    inners.push(lap);
+    if ((repeat === "\u239c" || repeat === "\u239f") && repeatCount === 0) {
```

Comment:
Why are parentheses special here? It seems like it would be a good idea to use a small connector whenever `repeatCount === 0`.

Perhaps there are some symbols where a vertical line segment isn't the right choice, but I feel like this might still be a better choice than the `else` case... Or at least perhaps we should do a little analysis of all the delimiters to decide?

Reply:
I don't get to pick any arbitrary small connector. If I don't use an SVG, then I am limited to font glyphs. And the selection of font glyphs is not ideal. The short ones were too thick, or had the wrong xMin distance or the wrong advance distance. After trying some alternative glyphs that made sense, none of them worked. I gave up and went the SVG route. It's a sure thing.

Reply:
Yes, that makes sense. I was wondering why we're fixing the case of parentheses but not fixing the case of e.g. curly braces (which use `\u23aa` for their `repeat` character). Would bad things happen if we replaced this `if` statement with
```js
 if (repeatCount === 0) {
```
?

Reply:
Curly braces have a longer straight section built into the glyph. I checked, and they do not need a correction similar to this PR. The existing short glyph remains hidden within the curly brace glyph.